sspl_test/functional_tests/alerts/node/test_node_bmc_interface.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+# -*- coding: utf-8 -*-
+
+from sspl_constants import BMCInterface
+from alerts.node import simulate_bmc_interface_alert
+from framework.base.functional_test_base import TestCaseBase
+
+from cortx.utils.process import SimpleProcess
+
+
+class BmcInterfaceTest(TestCaseBase):
+    def init(self):
+        pass
+
+    def filter(self, msg):
+        try:
+            # Make sure we get back the message type that matches the request
+            msg_type = msg.get("sensor_response_type")
+            if (
+                msg_type["info"]["resource_type"] == "node:bmc:interface:kcs"
```

Comment:
filtering should also happen based on bmc interface value extracted earlier, kcs or lan